docs/source/conf.py

```diff
@@ -1,26 +1,258 @@
+#!/usr/bin/env python
+#
+# pyesmda documentation build configuration file, created by
+# sphinx-quickstart on Fri Jun  9 13:47:02 2017.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+# If extensions (or modules to document with autodoc) are in another
+# directory, add these directories to sys.path here. If the directory is
+# relative to the documentation root, use os.path.abspath to make it
+# absolute, like shown here.
+
+import datetime
+
+# FIX: https://github.com/mgaitan/sphinxcontrib-mermaid/issues/72
+import errno
+import os
+import sys
 from subprocess import check_output
 
+import sphinx.util.osutil
+from sphinx.ext.napoleon.docstring import GoogleDocstring
+
+import iterative_ensemble_smoother as ies
+
+sphinx.util.osutil.ENOENT = errno.ENOENT
+
+
+package_path = os.path.abspath("..")
+sys.path.insert(0, package_path)
+
+
+def skip(app, what, name, obj, skip, options):
+    if name in ["__call__"]:
+        return False
+    return skip
+
+
+def setup(app):
+    app.connect("autodoc-skip-member", skip)
+
+
 project = "iterative_ensemble_smoother"
-copyright = "2022, Equinor"
 author = "Equinor"
-release = "0.1.1"
-
+copyright = f"2022-{datetime.datetime.today().year}, {author}"
+release = ies.__version__
+version = ies.__version__
 
+# convert the python file to a notebook
 check_output(["jupytext", "Polynomial.py", "-o", "Polynomial.ipynb"])
 
+# Do the same for this file
 check_output(["jupytext", "Oscillator.py", "-o", "Oscillator.ipynb"])
 
-
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
 extensions = [
```

Comment:
Anything here that we do not need? This list grew quite a bit.

Reply:
There were duplicates. I cleaned it.